.github/workflows/release.yml

```diff
@@ -60,18 +74,51 @@ jobs:
       - name: Calculate next version
         run: |
           echo "Latest tag: ${{ env.latest_tag }}"
-          IFS='.' read -r major minor patch <<< "${{ env.latest_tag }}"
-          new_minor=$((minor + 1))
-          new_tag="$major.$new_minor.0"
+          IFS='.' read -r major minor patch <<< "${env.latest_tag#v}"
+
+          if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
+            if [[ "${{ inputs.version_bump }}" == "patch" ]]; then
+              patch=$((patch + 1))
+            else
+              minor=$((minor + 1))
+              patch=0
+            fi
+          else
+            # Default behavior for scheduled runs
+            minor=$((minor + 1))
+            patch=0
+          fi
+
+          new_tag="v$major.$minor.$patch"
           echo "New tag: $new_tag"
           echo "new_tag=$new_tag" >> $GITHUB_ENV
 
-      # This will trigger a deploy via .github/workflows/cd.yml
-      - name: Push New Tag
+      - name: Create and Push Tag
         run: |
           git config user.name "github-actions[bot]"
           git config user.email "github-actions[bot]@users.noreply.github.com"
           git tag ${{ env.new_tag }}
           git push origin ${{ env.new_tag }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_API_TOKEN }}
+
+      - name: Setup Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: 1.22.x
+
+      - name: Run goreleaser
+        uses: goreleaser/goreleaser-action@v4
+        with:
+          distribution: goreleaser
+          version: v1.17.2
+          args: release --clean
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_API_TOKEN}}
+
+      - name: Bump Homebrew formula
+        uses: dawidd6/action-homebrew-bump-formula@v3
+        with:
+          token: ${{secrets.GITHUB_API_TOKEN}}
+          formula: lazygit
+          tag: ${{env.new_tag}}
```

Comment:
Passing the tag explicitly as per https://github.com/dawidd6/action-homebrew-bump-formula#standard-mode